platform/os_windows.go

```diff
@@ -130,33 +143,37 @@ func SetOutboundSNAT(subnet string) error {
 
 // ClearNetworkConfiguration clears the azure-vnet.json contents.
 // This will be called only when reboot is detected - This is windows specific
-func ClearNetworkConfiguration() (bool, error) {
+func (p *execClient) ClearNetworkConfiguration() (bool, error) {
 	jsonStore := CNIRuntimePath + "azure-vnet.json"
-	log.Printf("Deleting the json store %s", jsonStore)
+	p.logger.Info("Deleting the json", zap.String("store", jsonStore))
 	cmd := exec.Command("cmd", "/c", "del", jsonStore)
 
 	if err := cmd.Run(); err != nil {
-		log.Printf("Error deleting the json store %s", jsonStore)
+		p.logger.Info("Error deleting the json", zap.String("store", jsonStore))
 		return true, err
 	}
 
 	return true, nil
 }
 
 func KillProcessByName(processName string) {
-	p := NewExecClient()
+	p := NewExecClient(nil)
```

Comment:
isn't this function used by CNI..if so i would suggest to pass execClient as receiver to this api

Reply:
It's used by telemetry package and does not log anything here, so we can leave it here

Reply:
telemetry package is used by CNI..so the logger should be passed from there